packages/shared/lib/utils/telemetry.ts

```diff
@@ -44,10 +44,10 @@ class Telemetry {
     private logInstance: v2.LogsApi | undefined;
     constructor() {
         try {
-            if ((isCloud || isEnterprise) && process.env['DD_API_KEY'] && process.env['DD_APP_KEY']) {
+            if ((isCloud || isEnterprise) && process.env['DD_API_KEY'] && process.env['DD_APP_KEY'] && process.env['DD_SITE']) {
                 const configuration = client.createConfiguration();
                 configuration.setServerVariables({
-                    site: 'us3.datadoghq.com'
+                    site: process.env['DD_SITE']
```

Comment:
I'm not sure we enforce this value, I would just make sure it works or has a default value

Reply:
what do you mean by `enforce`?
btw it is an already existing env var that we are already using in other part of the codebase

Reply:
I meant be sure it's there but I forgot there was the if above 🤦🏻